decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -92,6 +92,11 @@ def source_paths
       class_option :dev_ssl, type: :boolean,
                              default: false,
                              desc: "Don't add Puma development SSL configuration options"
+      def webpacker_gemfile_entry
+        return [] if options[:skip_javascript]
+
+        GemfileEntry.version "shakapacker", "~> 6.5", "Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker"
```

Comment:
Why do we need to add this to the Gemfile in the first place if it's already a dependency?

Reply:
Because Rails 6 has a dependency to webpacker, and we will end up in a situation when we would install webpacker first ( as a result of rails intallation process), then attempt to change it to shakapacker. 
Adding this, will allow the installation process to use shakapacker directly from the start, avoiding further steps like patch the webpacker.yml, configs etc.

Reply:
It would be good if we didn't need to add this.

Is there some way we can avoid this, e.g. overriding some of the tasks that are trying to install the webpacker dependencies?